frontend/src/components/dashboard/ActivityRow.vue

```diff
@@ -59,7 +107,7 @@ export default {
   components: { CategoryChip, AvatarRow, TextAlignBaseline },
   mixins: [dateHelperUTCFormatted],
   props: {
-    scheduleEntry: { type: Object, required: true },
+    scheduleEntry: { type: Object, default: () => ({ _meta: { loading: true } }) },
```

Comment:
oh, cool new pattern to avoid the NPE's.
I would highlight this to other team members, that they can use that too